### DISCUSSION TITLE
Does not work over X11 forwarding via ssh

### DISCUSSION THREAD

**Discussion by philclifford** [quickemu-project/quickemu#1364] [Quickgui]
I know this is low-priority edge-case stuff but while testing I found after `ssh -X <big beast upstairs>` the remote quickgui fails to run on the local display.
```
~/src/quickgui$ make version 
VERSION is 1.1.9
BUILD_ROOT is ../build-package
~/src/quickgui$ ./build/linux/x64/release/bundle/quickgui 

** (quickgui:4086066): WARNING **: 11:52:42.234: Failed to start Flutter renderer: No available configurations for the given RGBA pixel format
```
----
For reference - it works fine remotely using X2Go (and probably other similar tools), and there is no difference whether the command is run on a terminal on the remote host via ssh or on a  "local" X11-forwarded terminal from the remote host.

----

## Remote system
OS: Ubuntu 21.10 x86_64 
Kernel: 5.13.0-22-lowlatency 
Uptime: 6 days, 2 hours, 26 mins 
Packages: 5680 (dpkg), 17 (flatpak), 88 (snap) 
Shell: bash 5.1.8 
Resolution: 1600x900 
WM: Metacity (Marco) 
Theme: Adwaita [GTK3] 
Icons: Adwaita [GTK3] 
Terminal: /dev/pts/4 
CPU: AMD Ryzen Threadripper 1920X (24) @ 3.500GHz 
GPU: AMD ATI Radeon RX 550 640SP / RX 560/560X 
Memory: 23943MiB / 31957MiB 

## local system
OS: Ubuntu MATE 21.10 x86_64 
Host: Latitude E6530 01 
Kernel: 5.13.0-22-generic 
Uptime: 5 days, 22 hours, 32 mins 
Packages: 3973 (dpkg), 43 (snap) 
Shell: bash 5.1.8 
Resolution: 1600x900 
DE: MATE 1.26.0 
WM: Metacity (Marco) 
Theme: Yaru-MATE-Pink-dark [GTK2/3] 
Icons: Yaru-MATE-Pink-dark [GTK2/3] 
Terminal: mate-terminal 
Terminal Font: Ubuntu Mono 13 
CPU: Intel i7-3540M (4) @ 3.700GHz 
GPU: Intel 3rd Gen Core processor Graphics Controller 
GPU: NVIDIA NVS 5200M 
Memory: 3805MiB / 7822MiB

**Comment by CT-6969** [comment]
maybe try changing the color layout.
